app/shared/video-player/common/ui/guesture/VideoGestureHost.kt

```diff
@@ -623,7 +626,29 @@ fun VideoGestureHost(
                         },
                     )
                     .ifThen(family.swipeToSeek) {
-                        swipeToSeek(seekerState, Orientation.Horizontal)
+                        var previousVisibility by remember { mutableStateOf(ControllerVisibility.Invisible) }
+                        swipeToSeek(
+                            seekerState,
+                            Orientation.Horizontal,
+                            onDragStarted = {
+                                previousVisibility = controllerState.visibility
+                            },
+                            onDragStopped = {
+                                if (!controllerState.alwaysOn) {
+                                    controllerState.visibility = previousVisibility
+                                }
+                            },
+                        ) {
+                            progressSliderState.run {
+                                if (totalDurationMillis == 0L) return@run
+                                val offsetRatio =
+                                    (currentPositionMillis + seekerState.deltaSeconds.times(1000)).toFloat() / totalDurationMillis
+                                previewPositionRatio(offsetRatio)
+                                if (previousVisibility == ControllerVisibility.Invisible) {
+                                    controllerState.setRequestProgressBarVisible()
+                                }
+                            }
+                        }
```

Comment:
`setRequestProgressBarVisible` in `onDragStarted`, and `cancelRequestProgressBarVisible()` in `onDragStopped` sounds better to me

See the exisiting `controllerState.setRequest` for an example.

Reply:
Sorry my Chinese IME is not working